examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads 
+--------------------
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a geopandas.GeoDataFrame object.
+"""
+
+import geopandas as gpd
+import pygmt
+
+# read shapefile data using geopandas
+gdf = gpd.read_file(
+    "http://www2.census.gov/geo/tiger/TIGER2015/PRISECROADS/tl_2015_15_prisecroads.zip"
+)
+
+# the dataset contains different road types listed in the RTTYP column,
+# here we select the following ones to plot:
+# Common name roads
+roads_common = gdf[gdf.RTTYP == "M"]
+# State recognized roads
+roads_state = gdf[gdf.RTTYP == "S"]
+# Interstates
+roads_interstate = gdf[gdf.RTTYP == "I"]
```

Comment:
Maybe just put the comments in-line here.
```suggestion
roads_common = gdf[gdf.RTTYP == "M"]  # Common name roads
roads_state = gdf[gdf.RTTYP == "S"]  # State recognized roads
roads_interstate = gdf[gdf.RTTYP == "I"]  # Interstate roads
```